### DISCUSSION TITLE
Support for "date" / "datetime" field in product editor

### DISCUSSION THREAD

**Discussion by leewillis77** [woocommerce/woocommerce#47530] [WooCommerce New Product Editor]
The new product editor should support entry of date/datetime fields, either as a single field type which the caller can specify as date or datetime or as two separate field types. There are many use-cases where plugins may wish to extend the product editor with date/datetime fields, for example:

- to specify a future launch date (ie - product is available for purchase now, but won't be shipped until dd/mm/yyyy)
- special offer dates (separate to WooCommerce's scheduled pricing changes)
- event dates

**Comment by mattsherman** [comment]
Thanks @leewillis77 -- I've created an issue for this: #47820

Closing this discussion -- we'll use the issue to track and further refine requirements.
